cmfpy/model.py

```diff
@@ -236,6 +243,36 @@ def factors(self):
         except AttributeError:
             raise NOT_FITTED_ERROR
 
+    def argsort_units(self):
```

Comment:
This looks good and is more concise than the `sort_neurons` function in `visual.py`. I'll set a note to remove the old function at some point.